Donation-platform/src/main/java/com/google/PersonalizedRecommendations.java

```diff
@@ -88,7 +100,12 @@
 
     // Gets charities from the HardCodedCharitiesAndTags class
     private Collection<Charity> getAllCharities() {
-        return Arrays.asList(HardCodedCharitiesAndTags.charities);
+        // // Using hard-coded charities:
+        // return Arrays.asList(HardCodedCharitiesAndTags.charities);
```

Comment:
please cleanup

Reply:
Created new method dedicated to accessing hardcoded charities for testing.